components/badges/badges.css

```diff
@@ -0,0 +1,65 @@
+.profile-badge {
+    position: absolute;
+    height: 1rem;
+    width: 1rem;
+    bottom: 0.5rem;
```

Comment:
same here use px for anything less than 1 rem

Reply:
will change that to px